reproducible/ubuntu/node_gen_tar.bzl

```diff
@@ -73,17 +73,23 @@ load(
     "docker_build",
 )
 
-def nodebootstrap_image(name, env=None):
+def node_gen_tar_rule(name, env=None):
+    docker_build(
+        name = "%s_builder" % name,
+        base = ":ubuntu_build",
+        entrypoint = [
+            "/mknodeimage.sh",
+            "%s" % name[len("node_"):].replace("_", "."),
```

Comment:
nit: can you just remove the `"%s" %` here?

Reply:
I did this originally but had an error.  It doesn't make sense that it's related though, will change back.